touchforms/backend/xformplayer.py

```diff
@@ -151,14 +152,14 @@ def load_form(xform, instance=None, extensions=None, session_data=None, api_auth
 
     try:
         session_data.get('additional_filters', {}).update({'use_cache': 'true'})
-        form.initialize(instance is None, CCInstances(session_data, api_auth, form_context))
+        form.initialize(instance is None, CCInstances(session_data, api_auth))
     except CaseNotFound:
```

Comment:
Don't think this is necessary anymore as we won't be calling to HQ for our case list